new-website/src/components/location/edit-form.jsx

```diff
@@ -21,15 +29,112 @@ const useStyles = makeStyles((theme) => ({
     margin: '0',
     '& > *': {
       margin: theme.spacing(1),
+      minWidth: 300,
       flex: 'auto',
     },
   },
   editorLabel: {
     alignItems: 'normal',
     paddingTop: '15px',
   },
+  flexNone: {
+    flex: 'none',
+    minWidth: 50,
+  },
+  smallField: {
+    minWidth: 50,
+  },
+  markerIcon: {
+    flex: 'none',
+    minWidth: 40,
+    width: 40,
+    height: 52,
+    marginLeft: theme.spacing(2),
+  },
 }))
 
+function sentryInfo(msg) {
+  Sentry.withScope(scope => {
+    scope.setLevel("info")
+    Sentry.captureEvent({ message: msg })
+  })
+}
+
+function sentryWarn(msg) {
+  Sentry.withScope(scope => {
+    scope.setLevel("warning")
+    Sentry.captureEvent({ message: msg })
+  })
+}
+
+function sentryFatal(err) {
+  Sentry.withScope(scope => {
+    scope.setLevel("fatal")
+    Sentry.captureException(err)
+  })
+}
+
+// Resolve coordinates for given address with LocationIQ
+async function requestLatLng(address) {
+  const sleep = async (ms) => {
+    await new Promise((wakeup, _) => setTimeout(wakeup, ms));
+  };
+  const tryResolve = async (address, retries) => {
+    const STATUS_CODE_RATE_LIMIT_EXCEEDED = 429
+    for (let i = 0; i < retries; i++) {
+      // TODO: Easy way to hide our key? (not urgent, it's a free account)
```

Comment:
Client side hiding of key is not possible ... if you really want to find out the final key then you just check the request we do in you browser. Either we "hide" it in a function (server side) or we just accept this risk

Reply:
Yes, makes sense. I will clean up this and other TODOs in a follow-up PR.